using-timescaledb/continuous-aggregates.md

```diff
@@ -248,6 +246,117 @@ hypertable with a continuous aggregate.
 
 ---
 
+### Advanced Topics [](advanced-usage)
+
+#### Continuous Aggregates using Integer-Based Time [](create-integer)
+
+Usually, continuous aggregates are defined on a [date/time-type](https://www.postgresql.org/docs/current/datatype-datetime.html) column, but it is
```

Comment:
Can you make this into a footnote-style reference please

Reply:
Done.